students/Ben_freeman/lesson09/pngdiscover.py

```diff
@@ -0,0 +1,22 @@
+import os
+
+
+big_list = []
+
+
+def dive(directory):
+    global big_list
```

Comment:
Making _big_list_ a global is not necessary, as I said, it's the same list everywhere.